XRTK-Core/Packages/com.xrtk.core/Inspectors/ControllerPopupWindow.cs

```diff
@@ -369,15 +358,24 @@ private void RenderInteractionList(SerializedProperty interactionList, bool useC
 
             GUILayout.EndHorizontal();
 
-            for (int i = 0; i < interactionList.arraySize; i++)
+            for (int i = 0; i < interactionProfilesList.arraySize; i++)
             {
-                var interaction = interactionList.GetArrayElementAtIndex(i);
-                var axisType = interaction.FindPropertyRelative("axisType");
-                var action = interaction.FindPropertyRelative("inputAction");
-                var inputType = interaction.FindPropertyRelative("inputType");
-                var invertXAxis = interaction.FindPropertyRelative("invertXAxis");
-                var invertYAxis = interaction.FindPropertyRelative("invertYAxis");
-                var interactionDescription = interaction.FindPropertyRelative("description");
+                var interactionProfileProperty = interactionProfilesList.GetArrayElementAtIndex(i);
+                var mappingProfile = interactionProfileProperty.objectReferenceValue as MixedRealityInteractionMappingProfile;
+
+                if (mappingProfile == null)
+                {
+                    continue;
+                }
+
+                var interactionMappingProperty = new SerializedObject(mappingProfile).FindProperty("interactionMapping");
+
+                var axisType = interactionMappingProperty.FindPropertyRelative("axisType");
+                var action = interactionMappingProperty.FindPropertyRelative("inputAction");
+                var inputType = interactionMappingProperty.FindPropertyRelative("inputType");
+                var invertXAxis = interactionMappingProperty.FindPropertyRelative("invertXAxis");
+                var invertYAxis = interactionMappingProperty.FindPropertyRelative("invertYAxis");
```

Comment:
I could probably just outright remove these two to fix the popout window

CC @FejZa